src/utils/Booking.utils.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from 'react-bootstrap'
+import { Link } from 'react-router-dom'
+
+export function statusSelector(status: string, component: string, locationId: string) {
```

Comment:
if can write the example of Input and Output example for reuse in further.(Or use by another)It's better if you need to debug it later

Reply:
Okay. kab readability is important ! thanks